src/core/email-client.js

```diff
@@ -53,41 +68,154 @@ function getFreshImap (cb) {
 }
 
 class EmailClient {
-    constructor () {
+    //constructor takes the full name of user as String
+    constructor (name) {
         var self = this;
+        self._name = name; 
+        self._email = "";
```

Comment:
The email can change too (try to think of all possible use cases with this client) so would be a good idea to add a method to change email credentials